Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -907,7 +962,13 @@ define([
 
             ++tileProvider._usedDrawCommands;
 
-            command.debugShowBoundingVolume = (tile === tileProvider._debug.boundingSphereTile);
+            if (tile === tileProvider._debug.boundingSphereTile) {
+                if (defined(surfaceTile.orientedBoundingBox)) {
+                    createDebugOrientedBoundingBox(surfaceTile.orientedBoundingBox, Color.RED).update(context, frameState, commandList);
```

Comment:
Since (at least currently) there is only one debug primitive rendered at once, this seemed easier since I wouldn't have to muck around inside some tile class. This should avoid recreating it at least between frames while one tile is selected.